interface/src/transaction.rs

```diff
@@ -0,0 +1,21 @@
+pub use parity_scale_codec::{Decode, Encode};
```

Comment:
nit: I'd rather this file be called `payload.rs` since the term transaction is a bit confusing here